src/dashboard_compiler/panels/charts/xy/compile.py

```
@@ -64,13 +68,14 @@ def compile_series_type(chart: LensXYChartTypes | ESQLXYChartTypes) -> str:
     return series_type
 
 
-def compile_xy_chart_visualization_state(
+def compile_xy_chart_visualization_state(  # noqa: PLR0913
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)